app/src/main/res/layout/resume_lesson_fragment.xml

```diff
@@ -64,7 +64,7 @@
         android:layout_marginTop="16dp"
         android:layout_marginEnd="32dp"
         android:ellipsize="end"
-        android:textColor="@color/component_color_resume_lesson_activity_description_color"
+        android:textColor="@color/component_color_shared_primary_text_color"
```

Comment:
This should be replaced with `component_color_shared_secondary_1_text_color`

Reply:
In all other layouts too for this fragment.

Reply:
Done.